src/unix/dependencies/nuget.template.config

```diff
@@ -1,24 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) Microsoft Corporation. All rights reserved.
-     Licensed under the MIT License. -->
-<configuration>
-  <packageSources>
-    <clear />
-    <add key="InstrumentationEngine" value="https://devdiv.pkgs.visualstudio.com/_packaging/InstrumentationEngine/nuget/v3/index.json" />
```

Comment:
>InstrumentationEngine [](start = 14, length = 21)

Not 100% sure what we need from this internal feed

Reply:
The only package that might be consumed is the microbuild signing plugin, however currently we don't sign our Linux bits and running the SigneBuild succeeds. If we ever need that feed (since signing is internal to Microsoft), I can re-add it in the future.

---
In reply to: [338193403](https://github.com/microsoft/CLRInstrumentationEngine/pull/154#discussion_r338193403) [](ancestors = 338193403)